lib/lure/repo.go

```diff
@@ -183,3 +186,59 @@ func Execute(pwd string, command string, params ...string) (string, error) {
 
 	return out, nil
 }
+
+func cleanupUpdateBranches(auth Authentication, project Project, repo Repo) error {
+
+	if project.Vcs != Hg {
```

Comment:
we should close git branches also, finding the branches to close could be done in a generic way in hg.go and git.go no?